docs/content/docs/projects/CiSteps.md

```diff
@@ -1,52 +1,68 @@
 ---
-title: CI Gradle Plugin
+title: CI steps plugin
 type: docs
 ---
 
-# CI Gradle plugin
+# CI steps plugin
 
-Для настройки CI под конкретный Gradle модуль используется in-house плагин.
+{{< hint warning>}}
+Plugin wasn't tested outside Avito yet, so expect difficulties, or even blockers.
+However, if you interested, please [contact us]({{< ref "/docs/Contacts.md" >}})!
+{{< /hint >}}
 
-Решает задачу точечной настройки типа и строгости проверок для разных сценариев.
+Plugin creates chains of tasks for CI, encapsulating it under single gradle task.
+
+For example, scenario "Pull Request checks".\
+Typically, you need to create multiple CI builds and wire it into chain.\
+Chain configuration(and possibly parameters) leaks to CI server settings, and you have less control over its changes inside gradle.\
+It makes sense if resources of single machine limited, but if all chain could run effectively
+(e.g. tasks runs in parallel and server has extra resources) on single machine, having all configuration in Gradle has benefits.
+
+## Getting started
+
+Apply the plugin in the app's `build.gradle` file:
 
 ```groovy
 plugins {
     id("com.avito.android.cd")
 }
-
-builds {
-   ...
-}
 ```
 
-Применяется например в модуле приложения, однако нет никаких ограничений, чтобы применить плагин как-то иначе (см. `./build.gradle`)
+There are no restriction in applying plugin to root project, and it could make sense for "global repo checks"
```

Comment:
```suggestion
The plugin can be applied to a root project or any module.
```

I think we don't have to stress on restriction, but rather focus on user scenarios.